unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendRequestHUD/ISentFriendRequestHUDView.cs

```diff
@@ -18,5 +18,6 @@ public interface ISentFriendRequestHUDView
         void ShowPendingToCancel();
         void SetBodyMessage(string messageBody);
         void SetTimestamp(DateTime date);
+        void SetSortingOrder(int sortingOrder);
```

Comment:
I see this method in 2 different interfaces. Maybe it worth to present thin interface for it and go with Interface Segregation Principle? 
Also, this interface has `Dispose `method, but doesn't implement `IDisposable `interface. Can you please inherit it from `IDisposable`?

Reply:
You're right. Done in https://github.com/decentraland/unity-renderer/pull/3891/commits/856ff4b3e461c126a6d4c8cf42902f28fd6334dc. Thanks!